backend/src/common/mappers/page/map-page-to-ipage.ts

```diff
@@ -28,6 +29,15 @@ export const mapPageToIPage = (page: Page): IPage => {
     };
   });
 
+  const mappedPinnedUsers = pinnedUsers?.map((user) => {
+    return {
+      id: user.id,
+      fullName: user.fullName,
+      email: user.email,
+      avatar: user.avatar,
+    };
+  });
+
```

Comment:
Please try this example locally if it works 

```suggestion
  const mappedPinnedUsers = pinnedUsers?.map(({ id, fullName, email, avatar }) => {
    return {
      id,
      fullName,
      email,
      avatar,
    };
  });

```